### DISCUSSION TITLE
Master Database Block access and wait for recovery

### DISCUSSION THREAD

**Discussion by Xiaolefu93** [dotnetcore/FreeSql#1921] [General]
database:microsoft access database
Freesql Version:3.2.686

At the beginning, reading data was normal.In a few days, errors will start to occur when reading data.The error : Master Database Block access and wait for recovery. But Reconnection does not take effect,and the I restart the process does take effect

**Comment by 2881099** [comment]
new OledbConnection(connectionString).Open()

Test it
